etc/spack/defaults/cray/modules.yaml

```diff
@@ -13,9 +13,4 @@
 # Per-user settings (overrides default and site settings):
 #   ~/.spack/modules.yaml
 # -------------------------------------------------------------------------
-modules:
-  prefix_inspections:
-    lib:
-      - LD_LIBRARY_PATH
-    lib64:
-      - LD_LIBRARY_PATH
+modules: {}
```

Comment:
If these files are no longer being used they can just be removed

Reply:
That made the tests fail :( didn't check why, but then I thought it wouldn't hurt as a blueprint